pkg/yang/entry.go

```diff
@@ -89,6 +89,9 @@ type Entry struct {
 	RPC *RPCEntry // set if we are an RPC
 
 	Augments []*Entry // Augments associated with this entry
+
+	// Extra maps all the unsupported fields to their values
+	Extra map[string][]interface{}
```

Comment:
can't we do something alittle better than interface{} like at least type this to something reasonable or do we really not know what this can be?